<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractDialog</class>
 <widget class="QDialog" name="ExtractDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Extract Archive</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="archiveGroupBox">
     <property name="title">
      <string>Archive</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QLineEdit" name="archiveLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="archiveButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="fileCountLabel">
     <property name="text">
      <string>No archive selected</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputGroupBox">
     <property name="title">
      <string>Output Directory</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="outputDirLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="outputDirButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="overwriteCheckBox">
     <property name="text">
      <string>Overwrite existing files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="openFolderCheckBox">
     <property name="text">
      <string>Open folder after extraction</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExtractDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExtractDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
